docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -12,34 +18,35 @@ To use the component, simply combine it with the rest of your UI, e.g. in `setCo
 ```kotlin
 override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    
+
     // Load the data for the header here
-    
+
     setContent {
-        Column(Modifier.fillMaxSize()) {
-            MessageListHeader(
-                channel,
-                currentUser,
-                connectionState,
-                messageMode,
-                modifier = Modifier.fillMaxWidth(),
-                onBackPressed = { },
-                onHeaderActionClick = { },
-            )
-
-            // Rest of your UI
+        ChatTheme {
+            Column(Modifier.fillMaxSize()) {
+                MessageListHeader(
+                    modifier = Modifier.wrapContentHeight(),
+                    channel = channel,
+                    currentUser = currentUser,
+                    connectionState = connectionState,
+                    messageMode = messageMode,
+                    onBackPressed = { },
+                    onHeaderActionClick = { },
+                )
+
+                // Rest of your UI
+            }
         }
     }
-}
 ```
 
-This component doesn't have its own `ViewModel`, as it's stateless. You need to provide the data to the header, otherwise it doesn't know what to render.
+This component is stateless and therefore does not have it's own `ViewModel`. You need to provide the data to the header, otherwise it doesn't know what to render.
 
-We recommend using either our `ChatDomain` directly and combining information you get from it, or the `MessageListViewModel`, in pair with the rest of our components, for ease of use.
+We recommend using either our `ChatDomain` directly and combining information you get from it, or the `MessageListViewModel`. For greater ease of use you can use it with the rest of our components.
```

Comment:
We recommend using either our `ChatDomain` directly or the `MessageListViewModel` for the required state. For greater ease of use you can use it with the rest of our components.